### ISSUE TITLE
Android 7.0 cannot send data 

### ISSUE THREAD

**Issue by h124551753** [tradle/react-native-udp#46]
Android 7.0 cannot send data with IP both "255.255.255.255" and other IPs

**Comment by jsdario** [comment]
Seems it does not work on multicast, either

**Comment by flomair** [comment]
This issue is fixed by https://github.com/netbeast/react-native-udp. @jsdario maybe you can submit a pull request

**Comment by jsdario** [comment]
I believe in Netbeast we are currently using both, netbeast/react-native-udp and this project @v2.0.0 indistinguishbly.

Actually we found weird behaviour only when calling `setBroadcast` in Android 7 devices, which aparently is not necessary for it to work, *only in Android*, so in many ocasions you'll find code like the following:

```
if (Platform.OS !== 'android') {
  socket.setBroadcast(true)
}
```

This is however counter intuitive and, from my vary basic Linux knowledge, simply **wrong**. However it did the trick and sockets close normally. It is a huge workaround but I did not research further into the Java side to prove that actually our fork solves it. Initially it just changes how async threads are allocated and applies some styling.
